public/v4/apps/checkmk.yml

```diff
@@ -0,0 +1,33 @@
+captainVersion: 4
+services:
+    '$$cap_appname':
+        image: checkmk/check-mk-raw:$$cap_version
+        environment:
+            TZ: '$$cap_tz'
+            PHP_TZ: '$$cap_tz'
```

Comment:
I don't see these two env vars in checkmk's docs. Also the associated CapRover variable doesn't exist in the variables section.

Reply:
just some copy and paste errors. Fixed it.